src/components/composer/content_editable_helper.ts

```diff
@@ -1,9 +1,10 @@
 import { HtmlContent } from "./composer/composer";
 
 export class ContentEditableHelper {
+  // todo make el private and expose dedicated methods
```

Comment:
todo

Reply:
that 's a cherry pick of a saas-16.2 (i think) commit, the todo still stands to thins day and frankly not blocking, I'd rather keepit this way and address in master after the store imp